lighthouse-core/gather/connections/connection.js

```diff
@@ -106,11 +106,15 @@ class Connection {
           {method: callback.method, params: object.result}, 'verbose');
         return object.result;
       }));
+    } else if (object.id) {
+      const error = object.error && object.error.message;
```

Comment:
add a comment about why this might happen (pavel's old `"while notifications do not"` comment above probably should also be updated as it implies only two states)